src/canvas/Canvas.ts

```diff
@@ -402,7 +402,7 @@ export class Canvas extends SelectableCanvas implements CanvasOptions {
     this.targets = [];
     const target = this._searchPossibleTargets(
       this._objects,
-      this.getPointer(e, true)
+      this.getHTMLPointFromEvent(e)
```

Comment:
Weird, why do you use the HTML point to search the targets?

Reply:
Now it is visible that there are very strange usages and some buggy ones